setuptools/command/build.py

```diff
@@ -138,3 +143,4 @@ def get_output_mapping(self) -> Dict[str, str]:
         Destination files should be strings in the form of
         ``"{build_lib}/destination/file/path"``.
         """
+        ...
```

Comment:
Refer to https://github.com/microsoft/pyright/issues/6487

Reply:
I see the problem here is because we are talking specifically about prototypes, and the spec says:

> Bodies of protocol methods are type checked.

That is a bit disappointing... These pesky ellipsis can be annoying

Reply:
Further discussion: https://github.com/astral-sh/ruff/issues/8756#issuecomment-1817575511 . It does seem to be more accurate to spec. Ruff was also updated to not strip ellipses in Protocols in https://github.com/astral-sh/ruff/releases/tag/v0.1.6